### PR TITLE
Prepares for release of v2021.1 (CLI)

### PR DESCRIPTION
**Pull request by matsduf**
Updates:
* CLI.pm with new version (v3.1.0)
* Changes
* Makefile.PL (updated dependency)

~~This PR assumes that #205 will be merged before release.~~ DONE.

~~Travis will complain until https://github.com/zonemaster/zonemaster-engine/pull/924 is merged.~~ No, Travis logs

> Warning: prerequisite Zonemaster::Engine 4.002 not found. We have v4.1.1.

but it does not fail. With Github actions we get a failure.

### CODE REVIEW

**Comment by matsduf** [comment]
Travis logs
> Warning: prerequisite Zonemaster::Engine 4.002 not found. We have v4.1.1.
but it does not fail. With Github actions we get a failure.

**Comment by matsduf** [comment]
@mattias-p and @pnax, please review.

File: Changes [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
 
+v3.1.0 2021-05-28 (public release version)
+
+ [Features]
+ - Adds Finnish translation (PO file) (#192, #191)
+
+ [Fixes]
+ - Updates the installation instructions (#193, #191)
+ - Updates translations (#204, #201, #205, #200, #205,
+   #200, #207, #202)
```
Comment (pnax) [inline Changes:11]: Looks like you put #200 twice. I understand that the pattern is « _#pr_number, #issue_number_ », should we duplicate _#issue_number_ if it is listed more than once ?

**Review by pnax — DISMISSED** [review]
Looks good. Just a question on duplicated references

File: Changes (hunk 1 above)
Comment (matsduf) [reply Changes]: Good catch. It was "#205, #200" that came double. I added it first before it was merged, and then after. Corrected.

Comment (pnax) [reply Changes]: Oh you're right, I missed the duplication of both (good catch on my good catch ^^)

**Review by pnax — APPROVED** [review]

**Review by mattias-p — APPROVED** [review]

**Pull request merged**